src/platform/forms/tests/save-in-progress/actions.unit.spec.js

```diff
@@ -166,14 +167,14 @@ describe('Schemaform save / load actions:', () => {
           done(err);
         });
     });
-    it.skip('calls the Form 526-specific api to save the form', done => {
+    it('calls the Form 526-specific api to save the form', done => {
       const thunk = saveAndRedirectToReturnUrl(VA_FORM_IDS.FORM_21_526EZ, {});
       const dispatch = sinon.spy();
 
       thunk(dispatch, getState)
         .then(() => {
           expect(global.fetch.args[0][0]).to.contain(
-            '/v0/disability_compensation_in_progress_forms/21-526EZ',
+            inProgressApi(VA_FORM_IDS.FORM_21_526EZ),
```

Comment:
A week or two ago I was wondering why `inProgressApi()` existed because it seemed like it _always_ returned the same value. But I see it's now returning a different value for the 526 👍